src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -157,6 +159,8 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			publishEvent.withAdditionalMeasurements({
 				'elapsedTimeMs': new Date().getTime() - startTime
 			}).withAdditionalProperties(metadataTelemetryInfo).send();
+			// Close notification in 1 second.
```

Comment:
In case of error, I would not close the notification automatically so that user can see the details.

Reply:
Error seems to be handled in catch block - no?
I don't think this code will apply to error dialog, do you think so?

Reply:
actually you are right. in this case, why not close the notification in the finally block?

Reply:
my bad, the way the code is collapsed made me think this is in the same function...

Reply:
in both functions, you should add a finally block to close the notification regardless of the outcome.